docs/getting_started/setup_your_environment.md

```diff
@@ -45,16 +45,28 @@ or imports having unnecessary file extensions.
 
 The community has developed extensions for some editors to solve these issues:
 
+#### VS Code
+
 - [VS Code](https://marketplace.visualstudio.com/items?itemName=axetroy.vscode-deno)
   by [@axetroy](https://github.com/axetroy).
 
+#### JetBrains IDEs
+
 Support for JetBrains IDEs is not yet available, but you can follow and upvote
 these issues to stay up to date:
 
 - https://youtrack.jetbrains.com/issue/WEB-41607
 - https://youtrack.jetbrains.com/issue/WEB-42983
 - https://youtrack.jetbrains.com/issue/WEB-31667
 
+#### Vim
```

Comment:
Might be worth adding this works for both Vim and Neovim

Reply:
Updated. Thanks for the corrections.